### ISSUE TITLE
Companion Build Checking Script

### ISSUE THREAD

**Issue by AurevoirXavier** [darwinia-network/darwinia-common#470]
https://github.com/paritytech/substrate/blob/master/.maintain/gitlab/check_polkadot_companion_build.sh

**Comment by AurevoirXavier** [comment]
@wi1dcard Is possible to do this?

**Comment by wi1dcard** [comment]
Yes, it might be possible. However, I'll need to dig into it a little bit. Could we start this after the new year holiday?

**Comment by AurevoirXavier** [comment]
It's okay.

No rush.
